### PR TITLE
MGMT-7716: Add a template to deploy to console.redhat.com

### PR DESCRIPTION
**Pull request by carbonin**
## Description

This will be used to deploy the image service to the console.redhat.com
SaaS service. IMAGE_TAG is dynamically filled based on deployment
configuration.

## How was this code tested?

Deployed this template along with assisted-service on a crc cluster and successfully configured assisted-service to set infra-env download urls to the deployed image service.

Created an infra-env and successfully downloaded the image through a route I created pointing to image service service.

## Assignees

/cc @osherdp 
/cc @filanov 
/cc @rporres 

## Links

https://issues.redhat.com/browse/MGMT-7716

## Checklist

- [x] Title and description added to both, commit and PR
- [x] Relevant issues have been associated
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit tests (note that code changes require unit tests)

### CODE REVIEW

**Comment by osherdp** [comment]
/lgtm
consider adding ``PodDisruptionBudget`` as well. It made a great difference when we started incorporating it into assisted-service

**Comment by carbonin** [comment]
/hold

I'll add it here, forgot

File: deploy/template.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: ${PV_SIZE}
+- apiVersion: policy/v1
+  kind: PodDisruptionBudget
+  metadata:
+    name: assisted-image-service
+  spec:
+    minAvailable: 2
```
Comment (carbonin) [inline deploy/template.yaml:96]: Took this right from the assisted service one, do you think we need 2 here also?

Comment (carbonin) [reply deploy/template.yaml:96]: @osherdp

Comment (osherdp) [reply deploy/template.yaml:96]: yeah, should do no harm

**Pull request merged**